### ISSUE TITLE
CI: automatically publish to npmjs @next tag on push to master

### ISSUE THREAD

**Issue by vojtechsimetka** [ethersphere/bee-js#605]
This has been discussed on our roundtable on March 31st. Here is a dump of the proposal:

- rationale
    - we are anyways claiming that master always works
    - this truly gives the right weight to the `bee-js@next` idea and ppl (including us) being able to test it
- proposal
    - add ci action that does a pre-release to npmjs automatically on each merge to master
    - remove the manual pre-release mechanism

**Comment by AuHau** [comment]
Sounds good to me 👍 

I am just wondering if it does not make sense still to keep the pre-release mechanism? Maybe for "release candidates" testing?

**Comment by vojtechsimetka** [comment]
Yes but then probably it should be extended to have a functionality to provide specific commit to build from.

**Comment by Cafe137** [comment]
Closing, as a) it has been stale for a while, and b) releases are frequent enough so last release is only about 1 day behind master. We miss out on the advantage of being able to specify `@next` but there's been no requests for this in the community.
